tensorzero-core/src/clickhouse/migration_manager/mod.rs

```diff
@@ -42,6 +42,7 @@ use serde::{Deserialize, Serialize};
 /// This must match the number of migrations returned by `make_all_migrations` - the tests
 /// will panic if they don't match.
 pub const NUM_MIGRATIONS: usize = 28;
+const RUN_MIGRATIONS_DOCKER_COMMAND: &str = "docker run --rm -e TENSORZERO_CLICKHOUSE_URL=$TENSORZERO_CLICKHOUSE_URL tensorzero/gateway --run-migrations";
```

Comment:
Should we apply a version tag here, rather than use the latest image (in case someone is running an older gateeway)?

Reply:
good idea